src/pages/AboutUsPage/AboutUsPage.ts

```diff
@@ -0,0 +1,119 @@
+    });
+    const schoolLinkContainer = div({ className: styles.schoolLinkContainer });
+    const schoolLink = a({ href: 'https://rs.school/', target: '_blank', className: styles.schoolLink });
+    const schoolLogo = img({ className: styles.schoolLogo, src: './assets/icons/rsschool.png', alt: 'RS School logo' });
+    schoolLink.append(schoolLogo);
+    schoolLinkContainer.append(schoolLink);
+    this.pageWrapper.append(pageSecret, descriptionTeam, schoolLinkContainer);
+    return this.pageWrapper;
+  }
+
+  private createCardCV(developers: Developer[]): void {
+    const cardContainer = div({ className: styles.cardContainer });
+
+    developers.forEach((dev) => {
+      const devCard = this.createDeveloperCard(dev);
+      cardContainer.append(devCard);
+    });
+
+    this.pageWrapper.append(cardContainer);
+  }
+
+  private createDeveloperCard(dev: Developer): HTMLElement {
+    const devCard = div({ className: styles.devCard });
+
+    const photoContainer = div({ className: styles.photoContainer });
+    const photo = img({ className: styles.devPhoto, src: dev.photo, alt: `${dev.name}'s photo` });
+    photoContainer.append(photo);
+
+    const infoContainer = div({ className: styles.infoContainer });
+    const nameLink = a({ href: dev.github, target: '_blank', className: styles.devNameLink });
+    const nameElement = h3({ className: styles.devName, txt: dev.name });
+    const githubLogo = img({ className: styles.githubLogo, src: './assets/icons/github.png', alt: 'GitHub logo' });
+    nameLink.append(nameElement, githubLogo);
+    const roleElement = h4({ className: styles.devRole, txt: dev.role });
+    const educationElement = h4({ className: styles.devEducation, txt: dev.education });
+    const descriptionElement = p({ className: styles.devDescription, txt: dev.description });
+    const funFactElement = p({ className: styles.devFunFact, txt: `Note! ${dev.funFact}` });
+
+    infoContainer.append(nameLink, roleElement, educationElement, descriptionElement, funFactElement);
+    devCard.append(photoContainer, infoContainer);
+
+    return devCard;
+  }
+
+  private getDevelopers(): Developer[] {
```

Comment:
great implementation!   however it makes sense to move the data to separate file - as a constant or as a json and import it here. to split the content and rendering into separate entities